simde/arm/neon/get_high.h

```diff
@@ -65,9 +69,13 @@ simde_vget_high_f64(simde_float64x2_t a) {
     simde_float64x1_private r_;
     simde_float64x2_private a_ = simde_float64x2_to_private(a);
 
-    SIMDE_VECTORIZE
-    for (size_t i = 0 ; i < (sizeof(r_.values) / sizeof(r_.values[0])) ; i++) {
-      r_.values[i] = a_.values[i + (sizeof(r_.values) / sizeof(r_.values[0]))];
+    #if HEDLEY_HAS_BUILTIN(__builtin_shufflevector)
+      r_.values = __builtin_shufflevector(a_.values, a_.values, 1);
+    #else
+      #endif
```

Comment:
Oops.

Don't worry about it, I'll fix it when I merge (I always like to rewrite the commit message a little to include the bug number, so it's going to need a rebuild regardless).

Otherwise LGTM, thanks!

Reply:
oops indeed! thanks :)